data/common.yaml

```diff
@@ -0,0 +1,11 @@
+---
+python::version: '3'
+python::use_epel: true
+python::gunicorn_package_name: 'gunicorn'
+python::manage_pip_package: true
+python::manage_venv_package: true
+python::pip::group: 'root'
```

Comment:
Kudos to who can tell me what is wrong here: the default value is not catch by Automatic Parameter Lookup in `spec/defines/pip_spec.rb`.  Setting an explicit value in the test suite or a default value in `manifests/pip.pp` for the `$group` parameter workaround the issue, but I don't get why it is not working as expected.

Reply:
Also, setting values for sub-classes of python is unusual (and maybe not supported?) but I would like to avoid breaking backwards compatibility in this PR.